background/KF.ts

```diff
@@ -179,11 +179,18 @@ class Kee {
 
         this.networkAuth.startListening();
 
-        browser.privacy.services.passwordSavingEnabled.set({ value: false }, function () {
-            if (chrome.runtime.lastError != null) {
-                KeeLog.warn("KeeFox was unable to disable built-in password manager saving - confusion may ensue! " + chrome.runtime.lastError);
-            }
-        });
+        if (!configManager.current.enableBuiltInPasswordSaving)
+        {
+          browser.privacy.services.passwordSavingEnabled.set({ value: false }, function () {
+              if (chrome.runtime.lastError != null) {
+                  KeeLog.warn("KeeFox was unable to disable built-in password manager saving - confusion may ensue! " + chrome.runtime.lastError);
+              }
+          });
+        }
+        else
+        {
+          KeeLog.warn("we didn't try to disable built-in password manager saving");
```

Comment:
Please can we use this instead:

`KeeLog.debug("Kee didn't try to disable built-in password manager saving");`